### PR TITLE
Bug 1219645 - Implement *test_export_contacts_to_sdcard.py* as an int…

### PR DESCRIPTION
**Pull request by arcturus**
…egration test in JavaScript

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=01e1a9ce8724411f48c4e76f3828956b5578c388

File: apps/communications/contacts/test/marionette/export_sdcard_test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+var fs = require('fs');
+
+marionette('Export to SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
```
Comment (JohanLorenzo) [inline apps/communications/contacts/test/marionette/export_sdcard_test.js]: Defining the prefs here is not needed, there already are under `Contacts.config`.

File: apps/communications/contacts/test/marionette/export_sdcard_test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        'device.storage.testing',
+        'device.storage.prompt.testing'];
+  prefs.forEach(pref => {
+    profile[pref] = true;
+  });
+  var client = marionette.client({
+    profile: profile,
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
```
Comment (JohanLorenzo) [inline apps/communications/contacts/test/marionette/export_sdcard_test.js]: I'd put that under `Contacts.config` as this capabitilities is already defined in 8 other contacts test files.

File: apps/communications/contacts/test/marionette/export_sdcard_test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  });
+  var actions, subject, selectors;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
```
Comment (JohanLorenzo) [inline apps/communications/contacts/test/marionette/export_sdcard_test.js]: This line should be in `tearDown()`

**Pull request merged**